.gitignore

```diff
@@ -127,7 +127,6 @@ packages/web/app/next.config.mjs
 packages/web/app/environment-*.mjs
 packages/web/app/src/gql/*.ts
 packages/web/app/src/gql/*.json
-packages/web/docs/public/feed.xml
```

Comment:
next will put it inside `out` with migration to route handler `app/feed.xml/route.ts`